src/backend/distributed/transaction/remote_transaction.c

```diff
@@ -1291,7 +1291,7 @@ CheckRemoteTransactionsHealth(void)
  *
  * citus_<source group>_<pid>_<distributed transaction number>_<connection number>
  *
- * (at most 5+1+10+1+10+20+1+10 = 58 characters, while limit is 64)
+ * (at most 5+1+10+1+10+1+20+1+10 = 59 characters, while limit is 64)
```

Comment:
why do we have this change?

Reply:
Because all the underscore characters are included in the calculation by `+1`, except the one between `pid` and `distributed transaction number`. I though that it's missed. So added. Isn't it correct?